### ISSUE TITLE
Difference between "package" and "library" 

### ISSUE THREAD

**Issue by maneesha** [datacarpentry/python-ecology-lesson#600]
### What is the problem?

In this section (https://datacarpentry.org/python-ecology-lesson/00-before-we-start.html#asking-for-help) we say that a package is not the same as a library.  I know many people (myself included) tend to use these terms interchangeably.  This lesson even does so in itself -- mentioning the "pandas package" and the "pandas library" at different points.

**Comment by carlosmr12** [comment]
I don't think that is a good statement to have on that specific section of the lesson. I agree that package is not the same as a library, but in my opinion that might be too nuanced and potentially confusing. It would be a big ask that people new to python and seeking help should know what the difference is between these two concepts. My suggestion would be:

1.  Keep the consistency in the lesson (i.e. if pandas is a package, always refer to it as "pandas package")
2.  Replace that paragraph with something else (i.e. "Try to use specific terms and details to describe your problem. For instance, instead of saying 'my code isn't working,' explain what you expected to happen and what actually happened. Most people will understand vague descriptions, but being precise makes it easier for others to help you. The key point is to provide clear and detailed information about your issue.")

**Comment by maneesha** [comment]
Agreed, especially with the first point, because otherwise the lesson does not model the behavior it recommends.
